content/2_Bronze/Intro_Sets.mdx

```diff
@@ -571,10 +615,9 @@ See the following snippet for an example:
 
 ```java
 Map<Integer, Integer> m = new TreeMap<>();
+// m starts as {0: 0, 1: 1, 2: 2}
+m.put(0, 0); m.put(1, 1); m.put(2, 2);
```

Comment:
statements each on own line

Reply:
is there a need? i feel like this saves space and leaves room for the more important stuff. it also makes sense to group the operations on one line

Reply:
hm
good point, actually